app/views/comments/_comments.html.erb

```diff
@@ -6,18 +6,18 @@
         <th><i data-toggle="tooltip" title="Commenting method" class="fa fa-envelope"></i></th>
         <th><%= translation('users.list.comment') %></th>
         <th><%= translation('users.list.last_activity') %></th>
-
+        <a href="/profile/<%= comment.author.username %>"> <%=raw strip_tags(sanitize(RDiscount.new(comment.author.username).to_html)).truncate(150) %></a>
```

Comment:
Hi @Manaswee03,
Great work!

Just a few pointers:
-Adding lines 9 and 15 were not mentioned as part of the PR. You can remove these.

@TildaDares please correct if I am wrong

Reply:
Thankyou @TildaDares and @hsadia538 for your help.. I will surely try implementing your tips and get the work done at the earliest.